.travis.yml

```diff
@@ -8,21 +8,28 @@ php:
     - hhvm
 
 matrix:
+    fast_finish: true
     include:
         - php: 5.3
-          env: deps=low
+          env: COMPOSER_FLAGS="--prefer-lowest"
+        - php: 5.6
+          env: SYMFONY_VERSION=2.3.*
+        - php: 5.6
+          env: SYMFONY_VERSION=2.7.*
+        - php: 5.6
+          env: SYMFONY_VERSION=2.8.*@dev
+        - php: 5.6
+          env: SYMFONY_VERSION="3.0.x-dev as 2.8"
     allow_failures:
         - php: hhvm
+        - env: SYMFONY_VERSION=2.8.*@dev
+        - env: SYMFONY_VERSION="3.0.x-dev as 2.8"
 
-env:
-    global:
-        - deps=no
-
-before_install:
-    - composer self-update
+before_script:
+    - composer selfupdate
```

Comment:
To be sure to have the latest composer version, yes.

I can remove it if you want, but I would recommend to get always composer updated.